### PR TITLE
changed project architecture as planned.

### PR DESCRIPTION
**Pull request by tavisca-sgogawale**

### CODE REVIEW

File: src/OnlineRetailPortal.Contracts/Models/EntityStoreResponse.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OnlineRetailPortal.Contracts.Models
```
Comment (Tavisca-smohite) [inline src/OnlineRetailPortal.Contracts/Models/EntityStoreResponse.cs]: You can keep the namespace name consistent to onlineretailportal.contracts

File: src/OnlineRetailPortal.Web/Controllers/WeatherForecastController.cs [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+
+namespace OnlineRetailPortal.Web.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class WeatherForecastController : ControllerBase
```
Comment (Tavisca-smohite) [inline src/OnlineRetailPortal.Web/Controllers/WeatherForecastController.cs]: Do we need this

File: src/OnlineRetailPortal.Web/Models/WebRequest.cs [hunk 3]
```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace OnlineRetailPortal.Web.Models
+{
+    public class WebRequest
```
Comment (Tavisca-smohite) [inline src/OnlineRetailPortal.Web/Models/WebRequest.cs]: What these classes represent ? Change the name to something more meaningful

**Review by Tavisca-smohite — CHANGES_REQUESTED** [review]
Please remove Weather forcast related classes .
Also remove Empty class with name class1 .

File: src/OnlineRetailPortal.Web/Controllers/WeatherForecastController.cs (hunk 2 above)
Comment (tavisca-sgogawale) [reply src/OnlineRetailPortal.Web/Controllers/WeatherForecastController.cs]: removed WeatherForecastController
git does not push empty folder so I had created class1 is emptyclasss so git will push those empty folders to repo

**Review by achirametavisca — APPROVED** [review]
Looks good to me

**Review by Tavisca-smohite — APPROVED** [review]

**Pull request merged**